packages/wells/src/client/model/Well.ts

```diff
@@ -1,6 +1,7 @@
 import { Wellhead } from './wellhead';
 import { Wellbore } from './Wellbore';
-import { Asset } from 'wells/src/types';
+// import { Asset } from 'wells/src/types';
```

Comment:
why comment it out and not just remove it?

Reply:
ups, missed this one